anta/tests/cvx.py

```diff
@@ -87,3 +87,47 @@ def test(self) -> None:
         cluster_status = command_output["clusterStatus"]
         if (cluster_state := cluster_status.get("enabled")) != self.inputs.enabled:
             self.result.is_failure(f"Management CVX status is not valid: {cluster_state}")
+
+
+class VerifyActiveCVXConnections(AntaTest):
+    """Verifies the expected number of active CVX Connections.
+
+    Expected Results
+    ----------------
+    * Success: The test will pass if the expected number of connections is present.
+    * Failure: The test will fail if the unexpected number of connections is present.
+
+    Examples
+    --------
+    ```yaml
+    anta.tests.cvx:
+      - VerifyActiveCVXConnections:
+          expected_connection_count: 100
+    ```
+    """
+
+    categories: ClassVar[list[str]] = ["cvx"]
+    commands: ClassVar[list[AntaCommand | AntaTemplate]] = [AntaCommand(command="show cvx connections brief", revision=1)]
+
+    class Input(AntaTest.Input):
+        """Input model for the VerifyActiveCVXConnections test."""
+
+        expected_connection_count: int
```

Comment:
You could use PositiveInt here (need to import it https://github.com/aristanetworks/anta/blob/main/anta/custom_types.py#L170)
```suggestion
        connections_count: PositiveInteger
```